### ISSUE TITLE
Password input blocked on Mac

### ISSUE THREAD

**Issue by Ron1248** [albar965/littlenavmap#920]
**Software will not allow me to enter password on my Mac (Monterey 12.5.1) to connect to X-plane. 11**

"You have to enter your password in the next step since administrative
priviledges are required to fix the attributes.

Password:               _**The system does not allow any input on this line**_
Sorry, try again.
Password:
sudo: 1 incorrect password attempt

ERROR. Failed to change attributes.”

**Please don’t tell me I am doing something wrong.  

Is there a workaround?**

**Comment by albar965** [comment]
I suppose you run the script `.../Little Xpconnect/Prepare Little Xpconnect.command` according to https://www.littlenavmap.org/manuals/littlenavmap/release/2.6/en/INSTALLATION.html#clearing-the-quarantine-flag-for-little-xpconnect-on-macos
No idea why the password input fails.

You can also use this tool to clean up the quarantine flags:
https://forums.x-plane.org/index.php?/files/file/79828-mac-os-cleanup-quarantine-flags/
Probably easier to use and clears the quarantine flag for all X-Plane plugins.

**Comment by Ron1248** [comment]
Thank you for replying
Yes I ran the script and then opened the read me file.  No joy.
Also tried to use the clean up.
This has something to do with my Mac OS.  
Grrrrr

**Comment by albar965** [comment]
Does your login user account have the rights to administrate the computer?
![grafik](https://user-images.githubusercontent.com/15254956/188419587-5ca566cb-1fd0-41bd-bf33-08275c09c1f5.png)

**Comment by albar965** [comment]
Duplicate of #921

**Comment by jonaseberle** [comment]
I think the devs did indeed a very good job on this one. 
This has been a duplicate report so it was closed in favor of the other. And in the other several people tried to help but you were not answering their questions and being rude. Others could not reproduce your problem so it seemed too obscure to be solvable from this software's side anyways. You received much more support than your behaviour warrants.

**Comment by albar965** [comment]
> Good job guys. You can’t fix the problem so pretend it doesnt exist.

Sorry, but there are limitations. I cannot fix your computer or your OS installation if it breaks on basic things like password input.
